app/src/main/java/com/cepheuen/elegantnumberbuttonsample/MainActivity.java

```diff
@@ -1,96 +1,44 @@
 package com.cepheuen.elegantnumberbuttonsample;
 
 import android.graphics.Color;
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.util.Log;
-import android.view.View;
 import android.widget.TextView;
-
+import androidx.appcompat.app.AppCompatActivity;
 import com.cepheuen.elegantnumberbutton.view.ElegantNumberButton;
```

Comment:
Minor, adding a line break from previous imports is a good practice to isolate dependencies based on framework. It has zero impact on this code base though might be really helpful when the code base is huge.